<launch>
  <arg name="box_namespace" default="gt_box"/>
  
  <group ns="$(arg box_namespace)">
    <node pkg="grand_tour_ceres_apps" type="camera_camera_online_calibration" name="camera_camera_online_calibration" output="screen">
      <param name="capability_group" value="Calibration"/>
      <param name="initial_guess" value="$(find grand_tour_ceres_apps)/config/initial_guess.yaml"/>
    </node>
  </group> 
</launch>